### ISSUE TITLE
Improve documentation about 🚜s (tractors)

### ISSUE THREAD

**Issue by Hocuri** [kelseyhightower/nocode#4328]
There is no documentation about how to run this on :tractor:s (tractors), which contributed to lots of confusion, for example see #4292 (credit: reported by @songjiaqi1).

1. Can someone with more experience in this area add some context about how to run this code on :tractor:s?
2. I think that we should also add some general advice, because people may not know about the momentum of tractors, which may result in missed opportunities to save a life:
    - I don't know any specifics about the momentum of the :tractor: and @songjiaqi1's relative position (again, more context from someone who is more into this topic would be nice). But generally, in case it's currently moving towards you at more than, like, 5 m/s, you may want to make a step to the side, shortly before the :tractor: enters the spot where you are currently standing. I have never done this myself and would in no way consider myself an expert in this, but people who have done this usually report what a great feeling it is not to be brutally knocked over by a :tractor:.
    - The same is valid in case the :tractor: is hovering over your current position, even if it's not currently moving towards you. Tractors generally tend not to hover there for very long, so please consider stepping out of the way. Reportedly, even if the :tractor: is currently moving upwards, this may quickly change, and the :tractor: will stop falling upwards, starting to fall downwards. Again, no personal experience, but reportedly it's a great feeling to be in one piece.

**Comment by Hocuri** [comment]
Ping @justmert, do you know more about this?

**Comment by Hocuri** [comment]
Or maybe @rosesonfire, can you help with this? I'm currently digging in the dark about what we could do about this, any help appreciated :)

**Comment by justmert** [comment]
Thank you @Hocuri for bringing the attention to the most important topic for this project: running it on 🚜s (tractors)

Well, your two suggestions make sense because being one piece while a :tractor: moving towards us is the key part of the project. But i suggest another life saving algorithm that we can ship to the cloud (needs development, maybe aws or heroku i don't know)

What if we setup a ramp between us and :tractor: that while :tractor: coming towards us, we stand still like the solid philosophy of this project. If we give enough speed to the :tractor:s, it can definitely fly above and land behinds us. The problem is, how we are going to guarantee that :tractor: will enough speed to leap with ramp that all of us will be solid one piece. Any suggestion would be nice.

Also, we can set up wings for the :tractor: that it can fly above us longer time. Also needs cost and time, so feel free to discard it.

**Comment by justmert** [comment]
@Hocuri  Oh I have made a mistake, we are damaging to the project if we ship to the cloud. Forget about that, my bad.

**Comment by abhijit-hota** [comment]
Ah yes, this bug. 
I have come across an issue which wasn't specifically asking for running the code on a 🚜 but it had to do with something like that.

Issue: #4312 
See [my comment](https://github.com/kelseyhightower/nocode/issues/4312#issuecomment-772392685https://github.com/kelseyhightower/nocode/issues/4312#issuecomment-772392685) there for a solution.

I think if we resolve that, then it can be run on a 🚜 

Cheers

**Comment by Hocuri** [comment]
@justmert yeah, unfortunately, :tractor:s tend to fall downwards when stored in clouds; it's sad but I'm afraid we won't be able to ship to the cloud. (btw the other problem is that the same is true for ships; spaceships might work, but 1. we would still have the :tractor: problem and 2. spaceshipping to the cloud seems out of scope).

@abhijit-hota yes, resolving that issue would definitely make it work, thanks for the link to #4312! However, I'm slightly afraid that it might raise expectations (i.e. users will ask for this feature then).

Yet another idea: What about disrupting the :tractor: shortly before it hits us, using [DDD](https://github.com/joaomilho/Enterprise/#ddd---the-book) (Disruption Driven Development)? I know we would have to give up on our 0-bugs-promise, but what if we just introduce so many bugs that a single bug is not noticeable in the crowd?
